scripts/extractions/extract_meteo.py

```diff
@@ -0,0 +1,162 @@
+"""Extract AGERA5 (Meteo) data using OpenEO-GFMAP package."""
+import argparse
+from functools import partial
+from pathlib import Path
+
+import geojson
+import geopandas as gpd
+import openeo
+import pandas as pd
+from extract_sar import (
+    _buffer_geometry,
+    _filter_extract_true,
+    _pipeline_log,
+    _setup_logger,
+    _upload_geoparquet_artifactory,
+    create_job_dataframe,
+    generate_output_path,
+)
+from openeo_gfmap import Backend, TemporalContext
+from openeo_gfmap.backend import vito_connection
+from openeo_gfmap.manager.job_manager import GFMAPJobManager
+from openeo_gfmap.manager.job_splitters import (
```

Comment:
also here importing private methods. Can we avoid this?

Reply:
I changed them to non-private methods, except for some in GFMAP